vulcanai/utils.py

```diff
@@ -419,3 +430,163 @@ def stitch_datasets(df_list, on, index_list=None):
     merged_df.fillna(0, inplace=True)
```

Comment:
we have been filling Nans with -1 since 0 could be a valid answer for some questions